app/notifications/validators.py

```diff
@@ -147,17 +148,28 @@ def check_notification_content_is_not_empty(template_with_content):
         raise BadRequestError(message=message)
 
 
-def validate_template(template_id, personalisation, service, notification_type):
+def get_template_dict(template_id, service_id):
+    from app.schemas import template_schema
     try:
-        template = templates_dao.dao_get_template_by_id_and_service_id(
+        fetched_template = templates_dao.dao_get_template_by_id_and_service_id(
             template_id=template_id,
-            service_id=service.id
+            service_id=service_id
         )
     except NoResultFound:
         message = 'Template not found'
         raise BadRequestError(message=message,
                               fields=[{'template': message}])
 
+    return template_schema.dump(fetched_template).data
```

Comment:
Cool, so the template_schema is responsible for making sure `template.reply_to_text` has been worked out by calling `Template.get_reply_to_text`. Took me a little while to track that down